tests/modules/individuals/resources/test_individual_crud.py

```diff
@@ -0,0 +1,112 @@
+# -*- coding: utf-8 -*-
+# pylint: disable=missing-docstring
+from app.modules.individuals.models import Individual
+import logging
+import json
+
+from tests.modules.individuals.resources import utils as individual_utils
+
+from tests import utils
+
+log = logging.getLogger(__name__)
+
+
+def test_create_read_individual(flask_app_client, researcher_1, encounter_1):
+
+    # TODO add checks for is_public
+    encounter_1.owner = researcher_1
```

Comment:
encounter_1 is shared between tests and here you're changing it. This could cause problems if other tests use encounter_1 after his is run. Look at what Karen did with the mocking stuff as a good example of how this should be done

Reply:
I don't think I can use Mock here with database persistence and hanging the owner off the encounter. I'll make this safe thought so I'm not altering anything that will stick around.

Reply:
For the record, I did give it a shot!